test/support/conn_case.ex

```diff
@@ -25,4 +25,8 @@ defmodule VirtuosoWeb.ConnCase do
       @endpoint VirtuosoWeb.Endpoint
     end
   end
+
+  setup _tags do
+    {:ok, conn: Phoenix.ConnTest.build_conn()}
```

Comment:
This line was removed with Ecto but without it the controller tests will not work.